### ISSUE TITLE
torch.inductor error: Cannot create a file when that file already exists

### ISSUE THREAD

**Issue by generic-placeholder-name** [woct0rdho/triton-windows#10]
When I try to use `torch.compile`, the following error occurs: 

```
BackendCompilerFailed: backend='inductor' raised:
FileExistsError: [WinError 183] Cannot create a file when that file already exists: 'C:\\Users\\minhk\\AppData\\Local\\Temp\\torchinductor_minhk\\cache\\.27456.33436.tmp' -> 'C:\\Users\\minhk\\AppData\\Local\\Temp\\torchinductor_minhk\\cache\\2a71ae31166f4b5aef4ad99f88246ea08fca21f95829ec7d6819e72c196f217b'
```

I don't know what the problem is, since Triton itself actually works fine.

**Comment by rsxdalv** [comment]
Which versions is this fixed for? Pytorch 2.4.1 has this issue.

**Comment by woct0rdho** [comment]
It's in the main branch of torch but not released yet. Let's wait for torch 2.6

**Comment by rsxdalv** [comment]
Is it ok to keep the issue open meanwhile?
Also I wonder if they would ever release a 2.5.2 and 2.4.2 for such a reason. Although the compilation seems to still have a lot of issues just in general.

**Comment by woct0rdho** [comment]
Yes the compilation indeed has a lot of issues in general. I also encountered something like #18 and at this point I'm not sure how to solve this on most people's Windows without manually setting environments

**Comment by woct0rdho** [comment]
This should be fixed in PyTorch 2.6

**Comment by rsxdalv** [comment]
Appears to be fixed, thank you!
